benchmarks/test_sharded_parity.py

```diff
@@ -14,64 +14,48 @@
 from tests.base.boring_model import BoringModel, RandomDataset
 
 
-@pytest.mark.skipif(platform.system() == "Windows",
-                    reason="Distributed training is not supported on Windows")
-@pytest.mark.skipif(not _FAIRSCALE_AVAILABLE, reason="Fairscale is not available")
-def test_ddp_sharded_plugin_correctness_one_device():
```

Comment:
this test is not relevant for CPU with two processes, approved by @SeanNaren

Reply:
@Borda I don't see where `num_processes=2` in this test? Am I missing something?

Reply:
I have dropped this test as it was failing...
I have tested setting `num_processes=2` on master and it failed with the same error so it is related to num proc 2 not to the change/cleaning we did